Resources/Prototypes/Roles/Jobs/departments.yml

```
@@ -20,7 +20,7 @@
   - Clown
   - HeadOfPersonnel
   - Janitor
-#  - Lawyer # Corvax-IAA
+  - Lawyer # Corvax-IAA
```

Это тогда можно убрать
```suggestion
  - Lawyer
```